src/ophys_etl/modules/video/util_classes.py

```diff
@@ -0,0 +1,44 @@
+from ophys_etl.modules.video.utils import (
+    apply_downsampled_mean_filter_to_video)
+from ophys_etl.modules.median_filtered_max_projection.utils import (
+    apply_median_filter_to_video)
+from functools import partial
+import numpy as np
+
+
+class VideoModuleMixin(object):
+
+    def _get_supplemental_args(self) -> dict:
+        """
+        Parse self.args and return a dict like
+        {'quantiles': optional tuple of quantiles to use in clipping video
+         'spatial_filter': optional callable spatial filter to apply to video
+         'video_dtype': dtype of video
+        """
+        if self.args['upper_quantile'] is not None:
+            quantiles = (self.args['lower_quantile'],
+                         self.args['upper_quantile'])
+        else:
+            quantiles = (0.0, 1.0)
```

Comment:
Could you instead change the default to 0 and 1 instead of `None`? Then this logic wouldn't be needed.

Reply:
Done